hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionFileSystem.java

```diff
@@ -640,19 +640,26 @@ Path commitDaughterRegion(final HRegionInfo regionInfo)
   /**
    * Create the region splits directory.
    */
-  void createSplitsDir() throws IOException {
+  void createSplitsDir(HRegionInfo daughterA, HRegionInfo daughterB) throws IOException {
```

Comment:
It is unclear to me how this is related to the rest of the changes. Could you provide some insight?

Reply:
@Apache9 included this in the branch-2/master patch, including it here in the backport too.

Reply:
I believe this fixed an issue with a hbck test so precommit for the branch-2 version could be green.